triage/2025-01-14.md

```diff
@@ -0,0 +1,142 @@
+| All ❌✅ (primary)                 | 0.2% | [0.2%, 0.2%] | 3     |
+- Probably just noise, and if not, it's not really worth investigating.
+
+
+mir_transform: implement `#[rustc_force_inline]` [#134082](https://github.com/rust-lang/rust/pull/134082) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=336209eef13882bd1e211b24779584cb7ef911eb&end=b1a7dfb91106018f47ed9dc9b27aee1977682868&stat=instructions:u)
+
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | 0.3%  | [0.2%, 0.3%]   | 10    |
+| Regressions ❌ <br /> (secondary)  | 0.3%  | [0.2%, 0.3%]   | 15    |
+| Improvements ✅ <br /> (primary)   | -0.1% | [-0.1%, -0.1%] | 1     |
+| Improvements ✅ <br /> (secondary) | -0.3% | [-0.4%, -0.3%] | 2     |
+| All ❌✅ (primary)                 | 0.2%  | [-0.1%, 0.3%]  | 11    |
+- Looks like some regressions snuck back in since the last run - I'm unsure if something happened in a rebase to reintroduce them. 
+- The actual regressions are much smaller than the original perf run, and I don't think they're big enough to demand an investigation.
+
+
+Add #[inline] to copy_from_slice [#135384](https://github.com/rust-lang/rust/pull/135384) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=627513a764a696b59bb2679cc1731a197a088a1d&end=48a426eca9df23b24b3559e545cf88dee61d4de9&stat=instructions:u)
+
+| (instructions:u)                   | mean | range        | count |
+|:----------------------------------:|:----:|:------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | 1.1% | [1.1%, 1.1%] | 1     |
+| Regressions ❌ <br /> (secondary)  | -    | -            | 0     |
+| Improvements ✅ <br /> (primary)   | -    | -            | 0     |
+| Improvements ✅ <br /> (secondary) | -    | -            | 0     |
+| All ❌✅ (primary)                 | 1.1% | [1.1%, 1.1%] | 1     |
+- From the PR "Small chaotic effect is expected, small binary size improvements oddly line up with the effect I was going for. Seems unlikely they're connected though, I wouldn't be surprised to see a different list of affected scenarios after merge."
+
+
+Rollup of 4 pull requests [#135438](https://github.com/rust-lang/rust/pull/135438) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=2ae9916816a448fcaab3b2da461de754eda0055a&end=1ab85fbd7474e8ce84d5283548f21472860de3e2&stat=instructions:u)
+
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | 0.4%  | [0.4%, 0.4%]   | 1     |
+| Regressions ❌ <br /> (secondary)  | 1.2%  | [0.9%, 1.6%]   | 6     |
+| Improvements ✅ <br /> (primary)   | -     | -              | 0     |
+| Improvements ✅ <br /> (secondary) | -0.2% | [-0.2%, -0.2%] | 1     |
+| All ❌✅ (primary)                 | 0.4%  | [0.4%, 0.4%]   | 1     |
```

Comment:
This misses a comment, we can just link to https://github.com/rust-lang/rust/pull/135426.